package.json

```diff
@@ -134,9 +134,10 @@
     "moment": "2.17.1",
     "rc-calendar": "7.5.1",
     "rc-time-picker": "2.2.1",
+    "react": "15.4.1",
     "react-addons-css-transition-group": "15.4.2",
     "react-dom": "15.4.1",
-    "react": "15.4.1",
+    "rtl-detect": "1.0.0",
```

Comment:
[It's actually a tiny module](https://github.com/shadiabuhilal/rtl-detect/blob/master/lib/rtl-detect.js). I was just going to do a simple array lookup, but this module seemed to do some nice coverage of language & country in locale codes. I *could* probably just run our list of locales through it and capture the result as a hardcoded lookup hash